packages/jaeger-ui/src/components/TracePage/index.tsx

```diff
@@ -45,24 +45,43 @@ import { TUpdateViewRangeTimeFunction, IViewRange, ViewRangeTimeUpdate } from '.
 import { getLocation, getUrl } from './url';
 import ErrorMessage from '../common/ErrorMessage';
 import LoadingIndicator from '../common/LoadingIndicator';
-import { extractUiFindFromState } from '../common/UiFindInput';
+import { extractUiFindFromState, TExtractUiFindFromStateReturn } from '../common/UiFindInput';
 import * as jaegerApiActions from '../../actions/jaeger-api';
 import { getUiFindVertexKeys } from '../TraceDiff/TraceDiffGraph/traceDiffGraphUtils';
 import { fetchedState } from '../../constants';
 import { FetchedTrace, ReduxState, TNil } from '../../types';
-import { Trace } from '../../types/trace';
+import { Log, Span, Trace } from '../../types/trace';
 import { TraceArchive } from '../../types/archive';
 import { EmbeddedState } from '../../types/embedded';
 import filterSpans from '../../utils/filter-spans';
 import updateUiFind from '../../utils/update-ui-find';
 
 import './index.css';
+import TTraceTimeline from '../../types/TTraceTimeline';
+
+type TDispatchPropsTimelineViewer = {
```

Comment:
this type should exist somewhere that `VirtualizedTraceView` can import it, to be DRYer.

Reply:
perhaps `TTraceTimelineDispatchFns` in `../../types/TTraceTimeline`